src/main/webapp/app/exercises/modeling/manage/example-modeling/example-modeling-submission.component.html

```diff
@@ -183,12 +183,15 @@ <h3 class="mt-3">
                         <textarea class="form-control" rows="4" maxLength="2000" [(ngModel)]="assessmentExplanation" [disabled]="readOnly"></textarea>
                     }
                     @if (assessmentMode) {
-                        <jhi-unreferenced-feedback
-                            [(feedbacks)]="unreferencedFeedback"
-                            [readOnly]="readOnly"
-                            [addReferenceIdForExampleSubmission]="true"
-                            (feedbacksChange)="onUnReferencedFeedbackChanged($event)"
-                        />
+                        @if (result && result.id) {
+                            <jhi-unreferenced-feedback
+                                [(feedbacks)]="unreferencedFeedback"
+                                [readOnly]="readOnly"
+                                [addReferenceIdForExampleSubmission]="true"
+                                (feedbacksChange)="onUnReferencedFeedbackChanged($event)"
+                                [resultId]="result.id!"
```

Comment:
Is the non null assertion really needed here?
```suggestion
                                [resultId]="result.id"
```